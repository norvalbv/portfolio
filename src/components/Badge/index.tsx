import React, { ReactElement } from 'react';

type BadgeProps = {
  tag: string;
};

const Badge = ({ tag }: BadgeProps): ReactElement => {
  return (
    <div className="text-x m-2 w-max rounded-lg bg-accent-secondary px-4 py-0.5 text-dark-text">
      {tag}
    </div>
  );
};

export default Badge;
