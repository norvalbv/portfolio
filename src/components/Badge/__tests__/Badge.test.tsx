import React from 'react';
import renderer from 'react-test-renderer';

import Badge from '..';

describe('<Badge />', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Badge tag="Test Tag" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
