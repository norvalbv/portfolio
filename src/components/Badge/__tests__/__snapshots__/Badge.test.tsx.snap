// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> should render correctly 1`] = `
<div
  className="text-x m-2 w-max rounded-lg bg-accent-secondary px-4 py-0.5 text-dark-text"
>
  Test Tag
</div>
`;
